### ISSUE TITLE
"KeyError: 'in'" when referencing a 'path' parameter component

### ISSUE THREAD

**Issue by jeyoon** [kiwicom/schemathesis#612]
Hello! I've been trying to get this to work with an OpenAPI 3.0.0 schema, but have been running into an 'internal error' which I've narrowed down to my use of a component path parameter. The stripped down API I am reproducing this with is below, verified with https://editor.swagger.io/

I am running schemathesis with docker, which works **if the parameter is defined in-line**.
docker run kiwicom/schemathesis:stable run http://$IP/api/openapi.json

If I comment out the in-line definition and use the $ref, it crashes with the error seen below.
Using postman to make the call normally doesn't seem to run into any issues.

```
openapi: "3.0.0"
info: { title: "Blank API", version: "1.0" }
servers:
  - url: 'http://$IP/api'

paths:
  /{path_parameter}:
    get:
      operationId: modules.clients.temp_method
      parameters:
        - in: path
          name: path_parameter
          required: true
          schema: { type: string }
        # - $ref: '#/components/parameters/PathParameter'
      responses:
        '200': { description: OK }

components:
  parameters:
    PathParameter:
      in: path
      name: path_parameter
      required: true
      schema: { type: string }
```

```
An internal error happened during a test run
Error: Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/schemathesis/runner/__init__.py", line 245, in execute_from_schema
    yield from runner.execute()
  File "/usr/local/lib/python3.8/site-packages/schemathesis/runner/impl/core.py", line 100, in execute
    for event in self._execute(results):
  File "/usr/local/lib/python3.8/site-packages/schemathesis/runner/impl/solo.py", line 19, in _execute
    yield from self._run_tests(
  File "/usr/local/lib/python3.8/site-packages/schemathesis/runner/impl/core.py", line 126, in _run_tests
    for endpoint, test in maker(template, settings, seed):
  File "/usr/local/lib/python3.8/site-packages/schemathesis/schemas.py", line 79, in get_all_tests
    test = make_test_or_exception(endpoint, func, settings, seed)
  File "/usr/local/lib/python3.8/site-packages/schemathesis/_hypothesis.py", line 45, in make_test_or_exception
    return create_test(endpoint, func, settings, seed=seed)
  File "/usr/local/lib/python3.8/site-packages/schemathesis/_hypothesis.py", line 30, in create_test
    strategy = endpoint.as_strategy(hooks=hook_dispatcher)
  File "/usr/local/lib/python3.8/site-packages/schemathesis/models.py", line 285, in as_strategy
    return get_case_strategy(self, hooks)
  File "/usr/local/lib/python3.8/site-packages/schemathesis/_hypothesis.py", line 153, in get_case_strategy
    parameter, value, endpoint.get_hypothesis_conversions(location)
  File "/usr/local/lib/python3.8/site-packages/schemathesis/models.py", line 291, in get_hypothesis_conversions
    definitions = [item for item in self.definition.raw.get("parameters", []) if item["in"] == location]
  File "/usr/local/lib/python3.8/site-packages/schemathesis/models.py", line 291, in <listcomp>
    definitions = [item for item in self.definition.raw.get("parameters", []) if item["in"] == location]
KeyError: 'in'
```

**Comment by Stranger6667** [comment]
Hi @jeyoon 

Thank you for reporting this. Indeed it is a bug - resolved definitions should be used there

**Comment by Stranger6667** [comment]
@jeyoon 

The fix is released in `1.8.0`. Let me know if it works for your case

**Comment by jeyoon** [comment]
Seems to work fine now. Thank you for the fix!
